### PR TITLE
DumbString.lettersInCommon, one argument null, JUnit test case added.

### PR DESCRIPTION
**Pull request by Dylan-Brown**

### CODE REVIEW

**Comment by ChrisMurphyOnline** [comment]
Sorry, the String b should be a null String, not the String with the characters 'n', 'u', 'l', 'l'

File: DumbStringTest.java [hunk 1]
```diff
@@ -27,5 +27,14 @@ public void testLettersInCommonDoubleCount() {
 		int actual = DumbString.lettersInCommon(a, b);
 		assertEquals(expected, actual);
 	}
+
+	@Test
+	public void testLettersInCommonNullArgument() {
+		String a = "aa";
+		String b = "null";
```
Comment (nfallen) [inline DumbStringTest.java]: Did you mean for this string to be null or did you want it to have the literal value "null"?

Comment (Dylan-Brown) [inline DumbStringTest.java]: Whoops. Yeah, I'll fix that now

Comment (Dylan-Brown) [inline DumbStringTest.java]: Now it's fixed.

**Pull request merged**